src/pybind/mgr/cephadm/tests/test_cephadm.py

```diff
@@ -963,3 +964,31 @@ def test_upgrade_run(self, use_repo_digest, cephadm_module: CephadmOrchestrator)
                 assert image == 'image@repo_digest'
             else:
                 assert image == 'image'
+
+    @mock.patch("cephadm.serve.CephadmServe._run_cephadm")
+    def test_ceph_volume_no_filter_for_batch(self, _run_cephadm, cephadm_module: CephadmOrchestrator):
+        _run_cephadm.return_value = ('{}', '', 0)
+
+        error_message = """cephadm exited with an error code: 1, stderr:/usr/bin/podman:stderr usage: ceph-volume inventory [-h] [--format {plain,json,json-pretty}] [path]/usr/bin/podman:stderr ceph-volume inventory: error: unrecognized arguments: --filter-for-batch
+Traceback (most recent call last):
+  File "<stdin>", line 6112, in <module>
+  File "<stdin>", line 1299, in _infer_fsid
+  File "<stdin>", line 1382, in _infer_image
+  File "<stdin>", line 3612, in command_ceph_volume
+  File "<stdin>", line 1061, in call_throws"""
+
+        with with_host(cephadm_module, 'test'):
+            _run_cephadm.reset_mock()
```

Comment:
Probably not needed

Reply:
you mean`reset_mock()`? it is, otherwise `_run_cephadm.mock_calls` a few lines below will also contain the mock calls that were issued while adding the host. This means `_run_cephadm.mock_calls` would consists of about 7-8 additional calls that are unrelated to this test